test/fixtures/tables.copy.gdocsliceclip.json

```diff
@@ -6,5 +6,6 @@
   "dct": "kix",
   "ds": false,
   "cses": false,
-  "sm": "other"
+  "sm": "other",
+  "si": ""
```

Comment:
Looks like this is the only consequential change, although it seems surprising that this would only happen for one fixture and not all of them. Going to try running an update locally just to see.

Reply:
Welp, I *don’t* get this change when updating locally, and instead there are no meaningful changes at all. 😕

I think I’m going to cancel this PR; very odd.